scripts/restore

```diff
@@ -112,24 +115,27 @@ ynh_secure_remove --file=/tmp/diaspora.dump
 #=================================================
 # Bundle the ruby app
 #=================================================
-ynh_script_progression --message="Precompile assets..." --weight=200
+ynh_script_progression --message="Precompile assets..." --weight=20
```

Comment:
mmmhhh on my machine, the 200 is *really* justified :-) What is your machine?

Reply:
The number is more a way to assess the proportion that the process will take than the actual time... You can se explanation in the linter report:  https://ci-apps.yunohost.org/ci/job/525

Reply:
Ok but in this case, the other numbers becomes completely meaningless. For instance we still have above:

>  ynh_script_progression --message="Reinstalling rvm and ruby..." --weight=50

but the ruby installation time is actually a lot less than the precompiling of assets...

Reply:
Ok let's do:

- 20 for this one
- 10 for reinstalling rvm and ruby (btw this should not be rvm, because if I remember correctly we use rbenv now)
- 5 for all other steps > 1 

What do you think?